docs/src/running-validator/validator-reqs.md

```diff
@@ -47,7 +52,7 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 256 GB, or more
+  - 512 GB or more if `account-index` is used
```

Comment:
Can you make `account-index` an internal link to where it is explained in the docs what this is?